login.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+
+<html lang="es">
+<head>
+  <meta charset="utf-8">
+  <title>HTML & CSS Training</title>
+  <meta name="description" content="training">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body class="login-page">
+  <div class="login-wrapper">
+    <img src="assets/img/LogoWolox.png" alt="Wolox Logo" class="logo">
+    <form action="#" method="POST" class="login-form">
+      <section class="login-field">
+        <label for="email">Email</label>
+        <input id="email" name="email" type="email" placeholder=" " autocomplete="username" autofocus>
+      </section>
+      <section class="login-field">
+        <label for="password">Password</label>
+        <input id="password" name="password" type="password" autocomplete="current-password">
+      </section>
+      <a id="signin" class="login-btn filled-btn" href="/book-list.html">Login</a>
+      <hr class="divider">
+      <a class="login-btn outlined-btn">Sign up</a>
```

Comment:
Maybe you can move the Sign Up button outside of the form. That way you won't need the `hr` and use a border 😉